src/constants/url.js

```diff
@@ -1,6 +1,6 @@
 // Constants related to API endpoints
 export const URL_LINK = {
-    imageHostLink: 'https://backoffice-dev.residat.com',
+    imageHostLink: 'https://admin.residat.com',
```

Comment:
just show the url we will make sure to complete it it will. avoid errors

Reply:
please what do you mean by just show the url?

Reply:
dont complete the. links with. for media just use the link that was sent